core/monitor/auction.go

```diff
@@ -30,6 +30,7 @@ type AuctionState struct {
 	m                  *types.Market           // keep market definition handy, useful to end auctions when default is FBA
 	extension          *types.AuctionTrigger   // Set if the current auction was extended, reset after the event was created
 	extensionEventSent bool
+	maxDuration        *time.Duration
```

Comment:
maybe I've missed it but I don't see how this is handled in the snapshot, i.e. suppose this has been updated, a node loading from snapshot will not see this value.

Reply:
I don't think we have to do anything in the snapshots. When we restore the markets themselves, we propagate the initial network parameter, which will allow us to check whether or not the opening auction has gone on for too long. There's no real state being added here